src/components/Profile.js

```diff
@@ -0,0 +1,14 @@
+import "../styles/Profile.css";
+
+export const Profile = ({ profile }) => {
```

Comment:
굳이 이렇게 props 를 객체 형태로 받을 필요 없이 
아래와 같이 풀어서 받아도 될 것 같아요.
```suggestion
export const Profile = ({ profileImageSource, email }) => {
```

사용처에서는 아래와 같이 props 를 넘겨주면 될 것 같고요!
```jsx
<Profile {... userData} />
```